### DISCUSSION TITLE
Where to submit a PR for an ingest pipeline?

### DISCUSSION THREAD

**Discussion by abesinger** [Security-Onion-Solutions/securityonion#6248] [Unsupported Versions] [answered]
I've got an improvement to the syslog pipeline (#5251), I'd like to put in a PR.  It's not clear to me whether I should submit the PR to the SO repo, or to the Elastic/Filebeat repo?

**Comment by dougburks** [comment, marked as answer]
Assuming you're referring to https://github.com/Security-Onion-Solutions/securityonion/blob/dev/salt/elasticsearch/files/ingest/syslog, then you can submit to our dev branch per the guidelines here:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/CONTRIBUTING.md

**Reply by abesinger** [reply]
yup, that's what I had in mind. Thanks!
